CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# How To Contribute
+
+Thanks for reading our contribution guidelines.
+
+The following guidelines for contribution should be followed if you want to submit a pull request.
+
+## How To Prepare
+
+* You need a [GitHub account](https://github.com/signup/free)
+* Submit an [issue ticket](https://github.com/thomasgriffin/TGM-Plugin-Activation/issues) for your issue if there is not one yet.
+    * Describe the issue and include steps to reproduce if it's a bug.
+    * Include relevant version numbers.
+    * Additional screenshots or videos are often helpful.
+* If you are able and want to fix this, fork the repository on GitHub.
+
+## Make Changes
+
+We use the [git-flow](http://nvie.com/posts/a-successful-git-branching-model/) branching model. Before reporting a bug or new feature, please check the `develop` branch to see if it's already been addressed.
+
+* In your forked repository, create a topic branch for your upcoming patch. Usually this is based on the `develop` branch.
+    * For enhancements, name the branch according to the feature e.g. `feature/auto-activate`.
```

Comment:
Not all Git clients support using a slash in branch names, also, not everything is a feature